src/storage/object/copy.js

```diff
@@ -113,7 +113,7 @@ export const copyFile = async (config, env, daCtx, sourceKey, details, isRename)
   } finally {
     if (Key.endsWith('.html')) {
       // Reset the collab cached state for the copied object
-      await invalidateCollab('syncAdmin', `${daCtx.origin}/source/${daCtx.org}/${Key}`, env);
+      await notifyCollab('syncAdmin', `${daCtx.origin}/source/${daCtx.org}/${Key}`, env);
```

Comment:
I see this was the case before already, but we should probably align the capitalization of `syncAdmin` vs `syncadmin`